modules/_labs/core/react/lib/theming/README.md

```diff
@@ -142,16 +138,38 @@ const theme: PartialCanvasTheme = {
 </CanvasProvider>;
 ```
 
-### Nesting Theme Providers
+### Bidirectionality
+
+The `CanvasProvider` also provide support for bidirectionality, useful for RTL languages. The direction, part of the theme, is set using `ContentDirection.LTR` or  `ContentDirection.RTL`.
```

Comment:
Nit: `provides`

Reply:
I still think calling out how this works is important here.

`CanvasProvider` wraps your components with a `bdo` element that has the `dir` attribute set to the value of the theme direction. Styled components using the [Canvas `styled` function](https://github.com/Workday/canvas-kit/blob/master/modules/_labs/core/react/lib/theming/styled.ts) will have their styles automatically flipped if dictated by the closest theme object.